src/main/java/com/bigbookmarket/domain/Buy.java

```diff
@@ -0,0 +1,30 @@
+package com.bigbookmarket.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+
+public class Buy extends BaseTimeEntity{
```

Comment:
```suggestion
public class Buy extends BaseTimeEntity {
```

중괄호 { 앞에 스페이스 넣어주세요.

인텔리제이에서 저장키 누르면 자동으로 정렬되지 않나요?
저 원래 이런거 별로 신경 안 쓰는 사람인데 너무 특이하게 코딩하셔서 무슨 이유로 이렇게 하시는건지 궁금하네요...🧐